### PR TITLE
New UI

### PR DESCRIPTION
**Pull request by lucasinocente**
I'm using Bulma following the #37 layout

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+import "firebase/auth";
 
 import Messages from './containers/Messages/Messages';
 import ContactsList from './containers/Contacts/ContactsList';
 import Login from './containers/Login/Login';
 import Verify from './containers/Login/Verify';
-import NotFound from './NotFound/NotFound';
+import NotFound from './containers/NotFound/NotFound';
+
+import './App.css';
```
Comment (smkamranqadri) [inline src/App.js]: Why two files, css and scss?

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+import './App.scss';
   
 function App() {
+  useEffect(() => {
+    try {
+      firebase.auth().onAuthStateChanged(function(user) {
+        if (user) window.location.pathname = `messages/${user.uid}`;
```
Comment (smkamranqadri) [inline src/App.js]: We can use the router to navigate instead of window.location which we reload the application.

File: src/App.js (hunk 1 above)
Comment (lucasinocente) [reply src/App.js]: Good question

Comment (lucasinocente) [reply src/App.js]: I removed the file :)

File: src/App.js (hunk 2 above)
Comment (lucasinocente) [reply src/App.js]: I will work on that after finish the layout. I think more today and I can finish

Comment (lucasinocente) [reply src/App.js]: I will do this in the next branch

**Pull request merged**